src/container/srv_target.c

```diff
@@ -1989,9 +1989,14 @@ cont_snap_update_one(void *vin)
 	struct ds_cont_child	*cont;
 	int			 rc;
 
-	rc = ds_cont_child_lookup(args->pool_uuid, args->cont_uuid, &cont);
+	/* The container should be exist on the system at this point, if non-exist on this target
+	 * it should be the case of reintegrate the container was destroyed ahead, so just
+	 * open_create the container here.
+	 */
+	rc = ds_cont_child_open_create(args->pool_uuid, args->cont_uuid, &cont);
```

Comment:
Won't it cause race with container destroy? If it's a workaround for reintegration issue, why don't we ensure the vos container created before fetching snapshot in reint code path?